pymongo/uri_parser.py

```diff
@@ -122,12 +122,10 @@ def parse_host(entity, default_port=DEFAULT_PORT):
 _IMPLICIT_TLSINSECURE_OPTS = {
     "tlsallowinvalidcertificates",
     "tlsallowinvalidhostnames",
-    "tlsdisableocspendpointcheck",}
+    "tlsdisableocspendpointcheck"}
 
 # Options that cannot be specified when tlsInsecure is also specified.
-_TLSINSECURE_EXCLUDE_OPTS = (
-        {k for k in _IMPLICIT_TLSINSECURE_OPTS} |
-        {INTERNAL_URI_OPTION_NAME_MAP[k] for k in _IMPLICIT_TLSINSECURE_OPTS})
+_TLSINSECURE_EXCLUDE_OPTS = _IMPLICIT_TLSINSECURE_OPTS
```

Comment:
Is this needed now?

Reply:
Not really. I just left it in, in case the group of implicit options ever diverges from the list of options that cannot be specified concurrently with tlsInsecure. I'll remove it.